R-package/R/xgb.Booster.R

```diff
@@ -605,7 +604,7 @@ validate.features <- function(bst, newdata) {
 #' change the value of that parameter for a model.
 #' Use [xgb.parameters<-()] to set or change model parameters.
 #'
-#' The [xgb.attributes<-()] setter either updates the existing or adds one or several attributes,
+#' The \code{xgb.attributes<-} setter either updates the existing or adds one or several attributes,
```

Comment:
Is this reversed?

Reply:
Year. It did not look good in the rendered file.

Reply:
Actually, the link points to the current page. This is why I wanted to remove the link. Now, I have replaced \code{} by ``